index.html

```diff
@@ -87,6 +181,479 @@ <h1 class="mt-5 Girls Who Code</h1>
         </div>
       </div>
     </div>
+
+     <!-- Grid -->
+<div class="container">
+  <div id="mentors">
+  <div class="row">
+          <div class="heading">
+            <h1>Mentors</h1>
+          </div>
+  </div>
+  <div class="row">
+    <div class="col-lg-2.5">
+      <div class="thumbnail">
+      <div class="card" onclick="this.classList.toggle('flipped')">
+            <div class="front">
+              <p>
+                 Lorem ipsum dolor sit amet, consectetuer adipiscing 
+                 <strong>strong</strong>. Cum sociis natoque penatibus 
+                 mus. Donec quam felis, ultricies nec, pellentesque 
+              </p>
+            </div>
+            <div class="back">
+            </div>
+      </div>
+    </div>
+  </div>
+    <div class="col-lg-2.5">
+      <div class="thumbnail">
+      <div class="card" onclick="this.classList.toggle('flipped')">
+            <div class="front">
+              <p>
+                 Lorem ipsum dolor sit amet, consectetuer adipiscing 
+                 <strong>strong</strong>. Cum sociis natoque penatibus 
+                 mus. Donec quam felis, ultricies nec, pellentesque 
+              <p>
+            </div>
+            <div class="back">
+            </div>
+      </div>
+    </div>
+  </div>
+    <div class="col-lg-2.5">
```

Comment:
can you explain this col-lg-2.5 because bootstrap doesn't have this class and i can't see one in css

Reply:
Really very sorry sir for replying late!!! I didn't see this change request yet!!
Sir I wanted 5 grid thumbnails to come in a row . So I thought to try "col-lg-2.5" though I was too not very sure about it but as it worked so I thought might be its okay. 
Sir I have already made all required changes regarding this PR but am not able to submit it as the base website of Girls-who-Code has changed.
I have made the animation look as it was earlier with the grid containing flip cards also intact and as well corrected the footer position.

Reply:
Don’t all sir on github 😅
Make the changes for the new code base and send new pr. I’m closing this one